### PR TITLE
Listview new format

### PR DESCRIPTION
**Pull request by gdt050579**

### CODE REVIEW

File: AppCUI/src/Controls/ListView.cpp [hunk 1]
```diff
@@ -1479,261 +1485,64 @@ void ListView::OnUpdateScrollBars()
...
-    if (Context != nullptr)
-    {
-        WRAPPER->DeleteAllColumns();
-    }
+    if (index < WRAPPER->Columns.Count)
+        return { nullptr, 0U };
+    return { this->Context, index };
 }
+
+//bool ListView::DeleteColumn(uint32 columnIndex)
```
Comment (gheorghitamutu) [inline AppCUI/src/Controls/ListView.cpp:1495]: why are these commented out?

File: AppCUI/src/Controls/ListView.cpp [hunk 2]
```diff
@@ -1479,261 +1485,64 @@ void ListView::OnUpdateScrollBars()
...
-uint32 ListView::GetItemHeight(ItemHandle item)
+ListViewItem ListView::GetItem(uint32 index)
 {
-    return WRAPPER->GetItemHeight(item);
+    if (this->Context == nullptr)
+        return { nullptr, 0 };
+    if (index >= WRAPPER->Items.List.size())
+        return { nullptr, 0 };
+    return { this->Context, index };
+    // uint32 value;
```
Comment (gheorghitamutu) [inline AppCUI/src/Controls/ListView.cpp:1540]: these are also commented out

File: Examples/Layout/main.cpp [hunk 3]
```diff
@@ -146,16 +145,16 @@ class MainWin : public Window
         if (eventType == Event::ButtonClicked)
         {
             auto itm = lst->GetCurrentItem();
-            if (itm != InvalidItemHandle)
+            if (itm.Exists())
```
Comment (gheorghitamutu) [inline Examples/Layout/main.cpp:148]: maybe IsValid?

**Review by gheorghitamutu — APPROVED** [review]

File: AppCUI/src/Controls/ListView.cpp (hunk 1 above)
Comment (gdt050579) [reply AppCUI/src/Controls/ListView.cpp:1495]: To be used later :)

**Pull request merged**